docs/api/cases/cases-api-get-default-connector.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[cases-get-default-connector]]
+== Get default case connector API
```

Comment:
> == Get default case connector API

The original title was "get current connector", however that seemed less clear to me, since it seems like each case can have a different connector currently assigned to it.  Whereas this particular connector seems to be the one set at the highest level (in the "Edit external configuration" settings). If I've misunderstood, just let me know!

Reply:
This endpoint returns the configuration of cases which in turn has a default connector. A case configuration, at the moment, has two settings: the closure setting and the default connector. The API returns both.

Reply:
Thanks for the clarification! I've renamed this API to "get case configuration" instead (and the others to "set case configuration" and "update case configuration") to reflect this broader scope.

Reply:
I've also added the phrase "external connection details" in the introductory text to align with the UI button.